package.json

```diff
@@ -38,53 +38,60 @@
   "devDependencies": {
     "@ephox/bedrock-client": "^14.1.0",
     "@ephox/bedrock-server": "^14.1.2",
-    "@ephox/oxide-icons-tools": "^3.1.1",
+    "@ephox/oxide-icons-tools": "^4.0.0",
     "@ephox/swag": "^4.6.0",
     "@tinymce/eslint-plugin": "^2.3.0",
     "@tinymce/moxiedoc": "^0.3.0",
-    "@types/chai": "^4.3.3",
-    "@types/dompurify": "3.0.2",
+    "@types/chai": "^4.3.12",
+    "@types/dompurify": "3.0.5",
     "@types/prismjs": "^1.16.6",
-    "chai": "^4.4.1",
-    "chalk": "^4.1.0",
+    "chai": "^5.1.0",
+    "chalk": "^5.3.0",
     "changie": "^1.14.0",
-    "emojilib": "^2.4.0",
+    "emojilib": "^3.0.11",
     "eslint-plugin-only-warn": "^1.1.0",
-    "fast-check": "^2.23.0",
-    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "fast-check": "^3.16.0",
+    "fork-ts-checker-webpack-plugin": "^9.0.2",
+    "glob-parent": "^6.0.2",
     "grunt": "^1.5.3",
     "grunt-contrib-clean": "^2.0.0",
     "grunt-contrib-concat": "^2.0.0",
     "grunt-contrib-copy": "^1.0.0",
+    "grunt-contrib-cssmin": "^5.0.0",
     "grunt-contrib-symlink": "^1.0.0",
     "grunt-contrib-watch": "^1.1.0",
     "grunt-eslint": "^24.0.0",
     "grunt-nuget-pack": "^1.0.1",
     "grunt-shell": "^4.0.0",
+    "grunt-stylelint": "^0.19.0",
     "grunt-terser": "^2.0.0",
-    "grunt-webpack": "^5.0.0",
+    "grunt-webpack": "^6.0.0",
     "lerna": "^8.1.2",
-    "less-plugin-autoprefix": "^2.0.0",
+    "less": "^4.2.0",
+    "less-plugin-preset-env": "^1.2.6",
     "load-grunt-tasks": "^5.1.0",
     "moxie-zip": "0.0.4",
     "npm-run-all": "^4.1.5",
-    "patch-package": "^7.0.0",
+    "patch-package": "^8.0.0",
+    "postcss": "^8.4.35",
+    "postcss-preset-env": "^9.4.0",
```

Comment:
This should be a transitive dependency from stylelint? I don't think we use it directly?

Reply:
this was added to remove the warnings caused by `postcss-preset-env` during install

Reply:
That's not an acceptable reason. We always need to question weird stuff especially when it comes to package management. Please check this sort of thing in detail next time.

I dug in further and it's because `less-plugin-preset-env` has `postcss-preset-env` as a peer dependency. No packages specify it as a direct dependency.

Reply:
yes and `postcss-preset-env` requires `postcss` to function, and it doesn't seem to pull in as a peer dep automatically.